CONTRIBUTING.rst

```diff
@@ -57,19 +57,25 @@ Get Started!
 
 Ready to contribute? Here's how to set up `eggplant` for local development.
 
+If you are completely new to source code versioning using git, please search
+for a video explaining it. And ask for help getting a git tool set up on your
+machine.
+
 .. note:: We have decided to use the very conventional
           `a simple git branching model <https://gist.github.com/jbenet/ee6c9ac48068889b0912>`_.
           Read the guide to get a good introduction to Git workflows.
 
-e.g.::
+::
 
-    $ git clone git@github.com:USERNAME/eggplant.git
+    $ git clone git@github.com:kbhff/eggplant.git
```

Comment:
At least that's how I remember we discussed it earlier because kbhff was overflowing with weird branches.